Makefile

```diff
@@ -149,6 +150,8 @@ run-integration-tests:
 	$(MAKE) -C pkg/helm test
 	$(MAKE) -C samples/rest-server test
 	$(MAKE) -C manager run-integration-tests
+	$(MAKE) setup-cluster
+	$(MAKE) -C manager run-deny-by-default-test
```

Comment:
Integration tests do not use opa, thus any tests of opa policies have no effect.